website/_kb/user-guide/alerts/setting-up-an-alert.md

```diff
@@ -20,37 +20,34 @@ In the Alerts view, you'll see all existing alerts.
-2. Select the column you want to watch.
-3. Select the trigger type (greater/less than or equals to).
-4. Select your magic number:
+1. Select the query you want to trigger an alert by typing in the search box. Note that **queries with parameters won't work**.
+2. Select the column you want to watch in the `Value Column` field. That field's current value is displayed to the right.
+3. Select the trigger type (greater/less than or equals to) with the `Op` field.
+4. Type your desired threshold into the `Reference` field:
     ![](/assets/images/docs/gitbook/alerts_settings.png)
-5. Leave Rearm Seconds empty to get 1 alert when the status changes from Triggered to OK, enter a number to get an alert every time the query runs (by schedule) + the Rearm seconds value. 
+5. Leave `Rearm Seconds` empty to get 1 alert when the status changes from Triggered to OK. Or enter a number to be alerted every time the query runs.
 6. Save.
 7. Define alert destinations - email, [Slack]({% link _kb/user-guide/alerts/slack-alert-destination.md %}), Mattermost, HipChat and webhooks are supported.
 
 <img src="/assets/images/docs/gitbook/alert_destination.png" width="60%">
 
 ## Alert Status & Frequency
 
-The Alert Status is checked every time the query is executed - (alerts only work with scheduled queries).
+The Alert Status is checked every time the query is executed, but an Alert notification is sent only for scheduled queries.
 
 Alerts have 3 Status Types:
 
-* `TRIGGERED` - the value you set the alert for is triggered (if you set your alert to trigger when the value of "cats" is greater than 1500 as long as it's above 1500 your alert is triggered)
-* `OK` - the value you have set to trigger the alert is not reached for now (might happen after the alert was triggered or before it was ever triggered, if your "cats" value is now 1470 your alert will show as OK)
-* `UNKNOWN` - you should see this status once you have set your alert and it wasn't yet checked. To make your alert in the know, run the query it is linked to after setting the alert.
+* `TRIGGERED` - Your alert has been tripped because the current value of the desired field matches your criteria. For example, if you set your alert to trigger when the value of "cats" is greater than 1500, your alert will remain triggered as long as the query shows more than 1500 "cats".
+* `OK` - Your alert is not tripped as of the most recent query execution. This can happen after the alert has been triggered (in which case the status will change from TRIGGERED to OK) or before it was ever triggered. If your "cats" value is now 1470 your alert will show as OK.
+* `UNKNOWN` - You should see this status when you first set up your alert because it hasn't been checked yet. You can run the query to confirm that the Alert logic works. Manually running the query can change the Alert status but will not trigger a notification.
```

Comment:
```suggestion
* `UNKNOWN` - You should see this status when you first set up your alert because it hasn't been checked yet. You can run the query to confirm that the Alert logic works.
```

(there is no difference between running the query manually or on schedule)

Reply:
Also maybe worth mentioning that the Alert will switch to UNKNOWN state when the query has no data or no value in the column you use for reference.